catalog/transformer/catalog-transformer-pdf/src/test/groovy/ddf/catalog/transformer/input/pdf/PdfInputTransformerSpecTest.groovy

```diff
@@ -43,9 +53,18 @@ class PdfInputTransformerSpecTest extends Specification {
     }
 
     void setup() {
+        metacardType = new MetacardTypeImpl("pdf", [
+            new AssociationsAttributes(),
+            new ContactAttributes(),
+            new MediaAttributes(),
+            new DateTimeAttributes(),
```

Comment:
nit : It looks like DateTime and Location aren't used on the actual `MetacardTypeImpl`, so they don't need to be added to the mock one